frontend/@types/console/index.d.ts

```diff
@@ -46,6 +46,7 @@ declare interface Window {
     developerCatalogCategories: string;
     userSettingsLocation: string;
     consolePlugins: string[]; // Console dynamic plugins enabled on the cluster
+    projectAccessClusterRoles: string;
```

Comment:
@rottencandy isn't it should be an array of string?

Reply:
Got it. Never mind.

Reply:
As discussed in slack, this is JSON encoded, that's fine. Good that we now tested this together ;)